src/models/ToDoList.js

```diff
@@ -24,16 +24,17 @@ export class ToDoList extends BaseModelRDMS {
    * @return {promise}
    *         Contains an array with the ToDo List object and its ToDo's
    */
-  findByIdWithToDos(id) {
-    return this.findById(id).then((response) => {
-      if (response.length) {
-        return this.ToDo.findBy('todo_list_id', id).then((todos) => {
-          response[0].todos = todos
-          return response
-        })
+  findByIdWithToDos( id ) {
+    return this.findById( id ).then( ( response ) => {
+      if ( response.length ) {
+        const responeArray = [...response];
```

Comment:
Its a best practise to not override the parameters of a function, to keep it pure and to not have side-effects. In that case doesnt have any real benefit since it comes from `then` but `eslint` complaint, I didnt want to remove the eslint rule, since it is indeed useful, so I copied the array, maybe disabling the rule for that file makes sense, since for big arrays that [...] will be slow.